src/components/MAPI/apps/utils.ts

```diff
@@ -803,3 +804,28 @@ export function compareApps(
 
   return a.metadata.name.localeCompare(b.metadata.name);
 }
+
+export function formatYAMLError(err: unknown): string {
+  if (err instanceof YAMLException) {
+    interface IYAMLExceptionInternals extends YAMLException {
```

Comment:
The `YAMLException` type from the library doesn't actually expose these fields, but we know they're there 🤓